java/client/src/main/java/glide/api/models/configuration/ProtocolVersion.java

```diff
@@ -0,0 +1,10 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.configuration;
+
+/** Represents the communication protocol with the server. */
+public enum ProtocolVersion {
+    /** Use RESP2 to communicate with the server nodes. */
+    RESP3,
+    /** Use RESP3 to communicate with the server nodes. */
```

Comment:
These comments look backwards. Is this right? I think RESP2 comment should be paired with RESP2 enum variant.

Reply:
Oh thanks. Copied from python client where docs comes under the symbol, not above it.